roles/dtc/common/templates/ndfc_inventory/common/fabric_inventory_no_bootstrap.j2

```diff
@@ -7,8 +7,16 @@
 - seed_ip: {{ switch['management']['management_ipv6_address'] }}
 {% endif %}
   auth_proto: {{ MD['vxlan']['global']['auth_proto'] | default(defaults.vxlan.global.auth_proto) }}
+{% if switch['management']['username'] is defined %}
+  user_name: {{ switch['management']['username'] }}
```

Comment:
We should update the template to grab the `user_name` and `password` even if it is available.  Just keep this as `PLACE_HOLDER_USERNAME` and `PLACE_HOLDER_PASSWORD` but update it dynamically using the `get_credentials` action plugin at run time just like we do for the gobal username and password.